server/channels/api4/plugin_test.go

```diff
@@ -1155,7 +1190,10 @@ func TestGetRemotePluginInMarketplace(t *testing.T) {
 		res.WriteHeader(http.StatusOK)
 		json, err := json.Marshal([]*model.MarketplacePlugin{samplePlugins[0]})
 		require.NoError(t, err)
-		res.Write(json)
+		if _, err := res.Write(json); err != nil {
+			http.Error(res, "Failed to write response", http.StatusInternalServerError)
+			return
+		}
```

Comment:
Please just check the error using `require.NoError(t, err)`

Reply:
I missed that out!